iris_gpubench/nvidia_nsight/setup_nsights.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+# Set up system
+cat /proc/sys/kernel/perf_event_paranoid
+
+sudo sh -c 'echo 2 >/proc/sys/kernel/perf_event_paranoid'
```

Comment:
Running commands in a subshell using sudo is not ideal - we should run this through shellcheck at a minimum and refactor later